libs/platform/src/lib/components/table/table.component.ts

```diff
@@ -1165,6 +1165,12 @@ export class TableComponent<T = any> extends Table implements AfterViewInit, OnD
     private _onTableRowsChanged(): void {
         this._calculateVisibleTableRows();
         this._calculateCheckedAll();
+
+        if (this._shouldFocusAfterPageHasChanged) {
+            this._cd.detectChanges();
```

Comment:
could we use markForCheck here instead of detectChanges?

Reply:
It won't work.